CHANGES.txt

```diff
@@ -1,3 +1,7 @@
+V3.3.1:
+    Developers:
```

Comment:
There is no need to increse de version 3.3.0 will be the next. Move this to the 3.3.0 section please

Reply:
Okay